### PR TITLE
Feature forms

### PR DESCRIPTION
**Pull request by DejazmachMolla**
- The project should enable adding and editing users
- The project should specify correct input types
- Forms inputs should be labled
- The project should provide error information on the user edit page
- The code should be linted

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  def edit
+    @user = User.find(params[:id])
+  end
+
+  def update
+    @user = User.find(params[:id])
+    @user.update(user_params)
+
+    if @user.save
+      redirect_to new_user_path
```
Comment (maelfosso) [inline app/controllers/users_controller.rb]: If it's OK redirect to `edit_user_path`

File: app/views/users/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<%# <form action="/users" method="post" accept-charset="UTF-8">
```
Comment (maelfosso) [inline app/views/users/new.html.erb]: It's preferable to use Rails helpers `users_path`

File: app/views/users/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  <input type="text" id="username" name="user[username]"><br><br>
+  <label for="email">Email:</label>
+  <input type="text" id="email" name="user[email]"><br><br>
+  <label for="password">Password:</label>
+  <input type="password" id="password" name="user[password]"><br><br>
+  <input type="submit" value="Submit">
+</form> %>
+
+
+<%#= form_tag("/users", method: "post") do %>
```
Comment (maelfosso) [inline app/views/users/new.html.erb]: It's preferable to use Rails helpers `users_path`

**Review by maelfosso — CHANGES_REQUESTED** [review]
# Changes required :red_circle:

Hello,

Thank you for the job done so far.
Just few things to change. Kindly read the comments left below, please and update your code.

Once done, submit a new code review request

Thanks

Mael FOSSO

**Review by MarioDena — APPROVED** [review]
Hello Great work implementing the requested changes!

Project approved! 🎖️ 

Cheers!

**Pull request merged**